packages/core/src/components/Table/stories/TableSamples/TableSamples.stories.tsx

```diff
@@ -28,8 +28,14 @@ import {
 import { Delete, Lock, Unlock } from "@hitachivantara/uikit-react-icons";
 import { makeData, getColumns, AssetEvent } from "../storiesUtils";
 
-import { TableComplete } from "./TableComplete.stories";
-import TableCompleteRaw from "./TableComplete.stories?raw";
+import { TableComplete } from "./TableCompleteSample";
```

Comment:
Some information and table samples are missing:

- The main table sample is missing information: "A table gathers relational data. It displays..." 
- The following samples are missing from the main table page: no data, simple table, grouped rows, responsive table, list row.
- The complete table is missing: https://lumada-design.github.io/uikit/master/?path=/docs/visualizations-table--complete-story
- We have a "Table Samples" page where we have repeated samples included in the table hooks: Alternative Layout Story, Column Resize, etc